test/full.js

```diff
@@ -664,9 +664,7 @@ test('Uses dataType compare', function (t) {
 
     compareRun = false;
     var foo = new Foo({ silliness: 'you' });
-    t.assert(compareRun);
```

Comment:
I see no reason for a `compare` to run on the initial set. Since I've removed that compare above, this test was failing.